### PR TITLE
Streak Functionality

### PR DESCRIPTION
**Pull request by jakerogers2022**
## Trello Ticket :ticket: ##
https://trello.com/c/xDhU7Sjy/69-be-calculate-streak
## Summary :chart_with_upwards_trend: ##
Added Streak functionality
## Test plan :clipboard: ##
Working on android

### CODE REVIEW

File: components/CommonCompHabitChart.js [hunk 1]
```diff
@@ -85,6 +87,26 @@ const CommonCompHabitChart = ({ groupMembersData, groupMembersNames, groupColor
     }
     //console.log(userDataList)
 
+    //calculatestreak
+    var streakLength = 0;
+    if (userDataList.length > 0){
```
Comment (Peiyj) [inline components/CommonCompHabitChart.js:92]: Hmmmm I am wondering if what will happen if userDataList[i] has all 0s. Should we reset streak to 0?

**Review by Peiyj — COMMENTED** [review]
Just have one small comment. Almost there!

File: components/CommonCompHabitChart.js (hunk 1 above)
Comment (jakerogers2022) [reply components/CommonCompHabitChart.js:92]: This just checks that the list of arrays isn't empty. The streak should still reset to 0 if none logged any data

**Review by Peiyj — APPROVED** [review]
lgtm!!

**Pull request merged**